c++/src/kj/compat/http.h

```diff
@@ -298,6 +298,13 @@ class HttpHeaders {
 
   kj::Maybe<kj::StringPtr> get(HttpHeaderId id) const;
   // Read a header.
+  //
+  // We do not provide a convenience method to access an unindexed header by string name because
```

Comment:
Hmm, I'd phrase this a little differently since "indexed headers" is not a term we've used anywhere else in the documentation. How about:

> Note that there is intentionally no method to look up a header by string name rather than header ID. The intent is that you should always allocate a header ID for any header that you care about, so that you can `get()` it by ID. Headers with registered IDs are stored in an array indexed by ID, making lookup fast. Headers without registered IDs are stored in a separate list that is optimized for re-transmission of the whole list, but not for lookup.

Reply:
Thanks, I prefer this update as it's more descriptive.